src/profile.cpp

```diff
@@ -410,9 +410,18 @@ void Profile::refreshModStatus()
       } else {
         namesRead.insert(lookupName);
       }
-      unsigned int modIndex = ModInfo::getIndex(lookupName);
+      unsigned int modIndex = ModInfo::findMod(
```

Comment:
`findMod()` is a linear search. It's called in a loop, once for each mod, which will degrade refresh performance for large mod lists.